openstef/pipeline/create_forecast.py

```diff
@@ -41,10 +41,12 @@ def create_forecast_pipeline(
         DataFrame with the forecast
 
     """
+    pid = pj["id"] if pj.forecast_pid is None else pj.forecast_pid
```

Comment:
Good to add a small comment on what is happening here e.g.: Selecting alternative forecast model if it exists

Reply:
Maybe also good to make pid more specific, e.g. prediction_model_pid

Reply:
If the names of the variables are more precise it'll be clearer. I make that change too.